runtime/sema/type.go

```diff
@@ -238,6 +238,16 @@ func NewTypeAnnotation(ty Type) *TypeAnnotation {
 	}
 }
 
+// toString
+
+const ToStringFunctionName = "toString"
```

Comment:
I anticipate in the future we will have additional "built-ins" for all types, at which point it might make sense to have those stored in a `typeBuiltIns map[string][FunctionType]` or something of the like -- for now though this looks good!

Reply:
That's a good idea! I'll refactor this 👍

Reply:
Generalized this in 2c21e53